scripts/generator/graphql_generator/csharp/SDK/NativeWebCheckout.cs.erb

```diff
@@ -0,0 +1,31 @@
+namespace <%= namespace %>.SDK {
+    using System;
+
+    #if !SHOPIFY_MONO_UNIT_TEST
+    using UnityEngine;
+    #endif
+
+    class NativeWebCheckout: IWebCheckout {
```

Comment:
Would it make more sense to turn `IWebCheckout` into an abstract base class instead of an interface? At least that way we could flatten the class hierarchy a bit.